sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -40,10 +36,6 @@ import { DatabaseModule } from "@sims/sims-db";
     AuthModule,
     ZeebeModule.forRoot(),
     NotificationsModule,
-    MSFAAIntegrationModule,
-    ECertIntegrationModule,
-    FedRestrictionIntegrationModule,
-    DisbursementReceiptIntegrationModule,
     AppAESTModule,
     AppInstitutionsModule,
     AppStudentsModule,
```

Comment:
Why is AppSystemAccessModule not removed? do we still have left-overs?

Reply:
we have one more controller to be moved to integration, I will add a todo
![image](https://user-images.githubusercontent.com/77353155/210415706-fbd07e62-a31e-4104-8ccf-1fd4b56ef40f.png)